ieeeuoft/hackathon-template
# fixed some inventory page UI bugs

## Overview

- Resolves #IEEE-178
- Fixing a couple Inventory Page UI Bugs:
- the inventory grid changing sizes on load and search (now the width stays the same)
- Search bar & search icon alignment on mobile view (now fixed with flexbox)
- Increased margin on text when there are no items found
- changed text to say Loading when items are being loaded
- changed alignment of clear and apply filters buttons so they look good on wide screens


## Unit Tests Created

- none


## Steps to QA

- go to inventory page, use filters and make sure everything displays nicely, no misalignments or containers randomly changing width and the functionality is correct
- test the same thing with different screen sizes (very wide, mobile and tablet widths --> leverage your browser's dev tools)
